docs/src/tricks.md

```diff
@@ -27,12 +27,12 @@ With imagemagick, you can also dynamically generate wallpapers based on the sele
 Similarly, you can use a template image and repaint it for the current theme.
 
 ```nix
-{ pkgs, ... }:
+{ pkgs, lib, ... }:
 let
   theme = "${pkgs.base16-schemes}/share/themes/catppuccin-latte.yaml";
   wallpaper = pkgs.runCommand "image.png" { } ''
-    COLOR=$(${pkgs.yq}/bin/yq -r .palette.base00 ${theme})
-    ${pkgs.imagemagick}/bin/magick -size 1920x1080 xc:$COLOR $out
+    COLOR=$(${lib.getExe pkgs.yq} -r .palette.base00 ${theme})
+    ${lib.getExe' pkgs.imagemagick "convert"} -size 1920x1080 xc:$COLOR $out
```

Comment:
This has changed the program being used: should be `${lib.getExe pkgs.imagemagick}`